build/app.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
```

Comment:
The build folder still made it here despite being in the gitignore file, is this deliberate?

Reply:
Oops! I'll take it out now.

Reply:
Great!